src/world.rs

```diff
@@ -295,6 +297,10 @@ impl<'a> Join for &'a Entities {
     }
 }
 
+// TODO: Is this correct?
+#[cfg(feature="parallel")]
+impl<'a> ParJoin for &'a Entities {}
```

Comment:
Yeah, it should be fine. I just put that there to remind me to recheck it as I stopped coding pretty late.